Remotion/ObjectBinding/Web/UI/Controls/BocEnumValueImplementation/Rendering/BocEnumValueRenderer.cs

```diff
@@ -169,7 +169,10 @@ private ListControl GetListControl (BocEnumValueRenderingContext renderingContex
       var isDropDownList = renderingContext.Control.ListControlStyle.ControlType == ListControlType.DropDownList;
 
       if (isRadioButtonList)
+      {
         listControl.Attributes.Add (HtmlTextWriterAttribute2.Role, HtmlRoleAttributeValue.RadioGroup);
+        renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, CssClassThemed);
```

Comment:
okay, turns out we already add the CssClassThemed to the root element or the BocEnumValue, so please recheck and then we can drop the redundant css class here.